src/Symfony/Component/Validator/Resources/translations/validators.no.xlf

```diff
@@ -386,6 +386,10 @@
                 <source>This value is not a valid International Securities Identification Number (ISIN).</source>
                 <target>Denne verdien er ikke et gyldig International Securities Identification Number (ISIN).</target>
             </trans-unit>
+            <trans-unit id="100">
+                <source>This value should be a valid expression.</source>
+                <target>Denne verdien skal være et gyldig uttrykk.</target>
```

Comment:
Hey @riadh26

I see you submitted language PRs to Luxembourgish, Norwegian, Slovenian and Gallician. Do you speak all these languages or have you talked with natives about this? 

To use services like Google Translate is not sufficient. It will provide questionable translations like this one. 

I can of course be wrong, you may speak all these languages or my Norwegian is too poor and this is in fact a good translation.